students/tfbanks/Lesson04/Trigrams.py

```diff
@@ -1,5 +1,49 @@
-# Trigrams Exercise by tfbanks
+# Trigrams.py Exercise by tfbanks
 
-#!/usr/bin/env python3
+# !/usr/bin/env python3
 
-close_statement = input("Press Any key to exit")
+import random
+
+# Imports file, replace a handful of punctuation and returns, finds a place to start and end and then splits the words.
+with open('sherlock.txt', 'r') as import_file:
+    file = import_file.read().replace('\n', ' ').replace('""', ' ').replace('!', ' ').replace(';', ' ').replace('?', ' ').replace('"', ' ').replace(',', ' ').replace('.', ' ')
```

Comment:
avoid using `file` as var name, it shadows a built-in (see different syntax highlighting)